src/metabase/models/user.clj

```diff
@@ -330,9 +331,10 @@
   admins will receive an email right away."
   [new-user :- NewUser]
   (u/prog1 (insert-new-user! (assoc new-user :google_auth true))
```

Comment:
The `prog1` macro has `:style/indent` metadata which indicates it should be indented like `defn` (i.e. with the first arg on the same line and subsequent args indented by two spaces). So the previous indentation was actually correct here. Your editor probably doesn't parse the `:style/indent` metadata automatically, although it should work now if you're using Clojure-LSP since Braden [added it a couple months ago](https://metaboat.slack.com/archives/CKZEMT1MJ/p1675265762179199)

Reply:
I think Calva does use Clojure-LSP, though it doesn't recognize the macro here. Not sure why, is there a way to manually run Clojure LSP? I tried the `clj-kondo` command found [here](https://github.com/metabase/metabase/wiki/Migrating-from-Leiningen-to-tools.deps), but there wasn't any changes.